docs/full-node/devnet.md

```diff
@@ -120,3 +118,8 @@ export BLOCKFLOW_NETWORK_ID=2
 export BLOCKFLOW_PORT=22973
 export DB_NAME=devnet
 java -jar explorer-backend-x.x.x.jar
+```
+
+## Using Docker
```

Comment:
Since docker approach is the most convenient one for most devs, I would put this at the first place, WDYT?

Reply:
:+1: good idea